codecov.yml

```diff
@@ -19,8 +19,11 @@ coverage:
         base: auto
 
 ignore:
-  - "main.c"
-  - "tests/*"
+  - "tests/**/*"
+  - "src/main.c"
+  - "src/ui/*"
+  - "src/utils/*"
```

Comment:
is it sufficient to add only `src/utils/**/*`?

Reply:
In `f4caab081ccc286882b8a807f92d88da19a45bd4`,
I added `src/ui/**/*` and `src/utils/**/*` BUT codedov ignored ONLY files in `src/utils/wrapper`.
I think `src/utils/*` indecates files in `src/utils` and `src/utils/**/*` indecates files in "subdirectories" of `src/utils`.